internal/storage/service_test.go

```diff
@@ -2033,3 +2033,126 @@ func TestUpdateTask(t *testing.T) {
 		assert.DeepEqual(t, re, task)
 	})
 }
+
+func TestCreateDeletionRequest(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Creates a DeletionRequest", func(t *testing.T) {
```

Comment:
I think this test name should be "Returns a persistence error", or something similar.  

It would be nice to have a test that creates a deletion request too, but maybe this got lost when splitting up the PRs?

Reply:
Good point, it was just like that, quite misleading, but it was the same for workflows and tasks. Since these function are just wrappers to the persistence layer and we use a mock in the tests, I don't think they add a lot of value but I have added actual creation tests for the three entities.